src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -28,7 +28,7 @@ public function compile(Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('$this->env->getRuntime(\'Symfony\Bridge\Twig\Form\TwigRenderer\')->setTheme(')
+            ->write('$this->env->getRuntime(\'Symfony\Component\Form\FormRenderer\')->setTheme(')
```

Comment:
This will break if the registered runtime still is `Symfony\Bridge\Twig\Form\TwigRenderer`, right? (by only using the bridge, not the bundle, i.e not in a symfony full stack app)